pyDeltaRCM/init_tools.py

```diff
@@ -138,10 +138,10 @@ def import_files(self, kwargs_dict={}):
                     input_file_vars[k] = user_dict[k]
                 else:
                     raise TypeError(f'Input for {str(k)} not of the '
-                                    'right type in yaml configuration '
-                                    'file {self.input_file}. '
-                                    'Input type was {type(k).__name__}, '
-                                    'but needs to be {expected_type}.')
+                                    f'right type in yaml configuration '
+                                    f'file {self.input_file}. '
+                                    f'Input type was {type(k).__name__}, '
+                                    f'but needs to be {expected_type}.')
```

Comment:
well that's stupid --- I thought the appeal of the f string was the seamless formatting...